Froger/Gold.java

```diff
@@ -0,0 +1,42 @@
+package Froger;
+
+import javafx.scene.effect.ImageInput;
+import javafx.scene.image.Image;
+import javafx.scene.shape.Rectangle;
+
+import java.util.Random;
+
+public class Gold extends Rectangle{
+
+    private int gridX;
+    private int gridY;
+    private GridMap gridMap;
+    private Random rand;
+
+    public Gold(GridMap gridMap){
+        this.setHeight(30);
+        this.setWidth(30);
+        this.gridMap = gridMap;
+        this.rand = new Random();
+        changeLook();
+        addToGrid();
+        randomLocation();
+    }
+
+    public void randomLocation(){
+        this.gridX = rand.nextInt(gridMap.getColumnCount()-2);
+        this.gridY = rand.nextInt(gridMap.getRowCount()-2);
+        addToGrid();
+    }
+    private void addToGrid(){
+        gridMap.getChildren().remove(this);
+        gridMap.add(this, this.gridX, this.gridY);
+    }
+    private void changeLook(){
+        String source = "file:/home/michal/IdeaProjects/mojaApka/src/Froger/images/money.png";
```

Comment:
juz drugi raz wiedze ta sciezke do pliku, idealne rozwiazanie to trzymac takie sciezki w pliku ``.properties``, mniej idealne to w jakiej klasie jako publiczna stala ale na pewno nie powtarzanie tego, potem zmiana sciezki bedzie strasznie upierdliwa